### DISCUSSION TITLE
Need MouseEnter and MouseLeave event for RibbonButton

### DISCUSSION THREAD

**Discussion by rulai-yinqiang** [harborsiem/WindowsRibbon#21] [Ideas]
I need to set the RibbonButton image dynamicly when MouseEnter and MouseLeave a RibbonButton.

Or MouseHover event is also enough for me.

**Comment by harborsiem** [comment]
In Ribbon Framework there is no MouseEnter, MouseLeave, MouseHoover for most Ribbon controls. Only in GalleryControls (eg. RibbonDropDownGallery) with items you can get a Preview and CancelPreview-Event which is like a MouseEnter, MouseLeave.

**Reply by rulai-yinqiang** [reply]
This is great! Thanks.
